src/gentropy/method/window_based_clumping.py

```diff
@@ -166,7 +168,16 @@ def clump(
 
         Returns:
             StudyLocus: clumped summary statistics (without locus collection)
+
+            Check WindowBasedClumpingStepConfig object for default values
         """
+        # p-value default value will change in the next major release
+        if gwas_significance == 5e-8:
+            warnings.warn(
+                "Default value for gwas_significance will change in 2.0 release",
```

Comment:
"Default value for gwas_significance will change in next release"

Reply:
I have dropped it